VimeoNetworkingExample-iOS/VimeoNetworkingExample-iOS.xcodeproj/project.pbxproj

```diff
@@ -1216,6 +1214,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3D1A821E9248D03C2D131632 /* Pods-VimeoNetworkingExample-tvOSTests.release.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
I think we'll need to set the tvOS test target to `$(inherited) also. I ran into this when running pod install:
```bash
[!] The `VimeoNetworkingExample-tvOSTests [Release]` target overrides the `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` build setting defined in `../Pods/Target Support Files/Pods-VimeoNetworkingExample-tvOSTests/Pods-VimeoNetworkingExample-tvOSTests.release.xcconfig'. This can lead to problems with the CocoaPods installation
```